src/frequenz/sdk/actor/power_distributing/power_distributing.py

```diff
@@ -32,28 +30,23 @@
 
 class PowerDistributingActor(Actor):
     # pylint: disable=too-many-instance-attributes
-    """Actor to distribute the power between batteries in a microgrid.
+    """Actor to distribute the power between components in a microgrid.
 
-    The purpose of this tool is to keep an equal SoC level in all batteries.
-    The PowerDistributingActor can have many concurrent users which at this time
-    need to be known at construction time.
+    The actor can have many concurrent component managers.
+    For each component manager a bidirectional channel needs to be created
+    through which they can send and receive requests and responses.
```

Comment:
I think this needs some changes.  One instance of the PowerDistributingActor can handle one component category.  The component cagegory and type need to be specified at actor startup and it will setup the correct component manager based on the given cagegory.

There are no bidirectional channels anymore.  The PowerDistributingActor takes a receiver for receiving requests and a sender for sending results.

Reply:
Updated based on the suggestion